android/src/androidTest/java/bbct/android/common/navigation/test/NavigateUpFromBaseballCardListTest.java

```diff
@@ -29,6 +30,8 @@
 import java.util.List;
 
 import bbct.android.common.R;
+import bbct.android.common.test.DatabaseUtil;
+import bbct.android.common.test.rule.DataTestRule;
```

Comment:
Are the imports added with this commit used anywhere in this test? You can quickly clean up unused commits by pressing Alt-Ctrl-O (Windows/Linux) or Alt-Cmd-O (Mac).

Reply:
noted